src/phpDocumentor/Descriptor/Builder/PhpParser/MethodAssembler.php

```diff
@@ -45,11 +45,11 @@ public function __construct(ArgumentAssembler $argumentAssembler)
      *
      * @param ClassMethod $data
      *
-     * @return MethodDescriptor
+     * @return Method
      */
     public function create($data)
     {
-        $methodDescriptor = new MethodDescriptor();
+        $methodDescriptor = new Method();
```

Comment:
I take it that the Method element should still be refactored since it does not receive any constructor arguments (such as the FQSEN)?